### PR TITLE
Update Release Changelogs

### PR DESCRIPTION
**Pull request by minnieliu**
Updating Change log for next release

### CODE REVIEW

File: sdk/communication/Azure.Communication.Administration/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (2020-11-16)
 ### Added
```
Comment (DominikMe) [inline sdk/communication/Azure.Communication.Administration/CHANGELOG.md:4]: We had breaking changes and added LRO support right? Let's mention this here

**Review by DominikMe — COMMENTED** [review]
Thanks for taking this on @minnieliu 

Added a comment for the administration changelog

File: sdk/communication/Azure.Communication.Administration/CHANGELOG.md (hunk 1 above)
Comment (minnieliu) [reply sdk/communication/Azure.Communication.Administration/CHANGELOG.md:4]: @DominikMe  do you know who mostly worked on these changes? I'll reach out for more details.

Comment (minnieliu) [reply sdk/communication/Azure.Communication.Administration/CHANGELOG.md:4]: Nvm.. checking history for PRs thanks!

File: sdk/communication/Azure.Communication.Administration/CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,31 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (2020-11-16)
```
Comment (DominikMe) [inline sdk/communication/Azure.Communication.Administration/CHANGELOG.md:3]: nit: add empty line between ## and ###

At least my VS Code had actually complains whenever I forget that.

File: sdk/communication/Azure.Communication.Administration/CHANGELOG.md [hunk 3]
```diff
@@ -1,8 +1,31 @@
...
 ### Added
 - Support for mocking all client methods that use models with internal constructors
+- Added support for long-running operations. More detail under Breaking Changes.
+
+### Breaking
+- Support long running operations for purchasing phone numbers
```
Comment (DominikMe) [inline sdk/communication/Azure.Communication.Administration/CHANGELOG.md:9]: nit: end all sentences with a full stop.

Even the short "Renamed <> to <>." ones

File: sdk/communication/Azure.Communication.Chat/CHANGELOG.md [hunk 4]
```diff
@@ -1,8 +1,9 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (2020-11-16)
 ### Added
 - Support for mocking all client methods that use models with internal constructors
+- Added unit test for pagination
```
Comment (DominikMe) [inline sdk/communication/Azure.Communication.Chat/CHANGELOG.md:6]: Same here and in the other changelogs, let's add full stops at the end of sentences for the sake of consistency.

File: sdk/communication/Azure.Communication.Administration/CHANGELOG.md (hunk 3 above)
Comment (DominikMe) [reply sdk/communication/Azure.Communication.Administration/CHANGELOG.md:9]: second nit: let's be consistent whether we write "long-running" or "long running", lines 6 and 9

**Review by DominikMe — APPROVED** [review]
Thank you!

I've added some minor nit-picks using my strict proof reader eyes 👀

File: sdk/communication/Azure.Communication.Administration/CHANGELOG.md (hunk 2 above)
Comment (minnieliu) [reply sdk/communication/Azure.Communication.Administration/CHANGELOG.md:3]: Thanks @DominikMe Added the extra line

File: sdk/communication/Azure.Communication.Administration/CHANGELOG.md (hunk 3 above)
Comment (minnieliu) [reply sdk/communication/Azure.Communication.Administration/CHANGELOG.md]: Done

**Pull request merged**